public/apps/account/test/account-app.test.tsx

```diff
@@ -116,4 +122,50 @@ describe('Account app', () => {
       done();
     });
   });
+
+  it('Should not show tenant selection popup when multitenancy is disabled from security index', (done) => {
+    (getDashboardsInfoSafe as jest.Mock).mockResolvedValueOnce({
+      default_tenant: '',
+      multitenancy_enabled: false,
+    });
+
+    setupTopNavButton(mockCoreStart, mockConfig as any);
+
+    process.nextTick(() => {
+      expect(getSavedTenant).toBeCalledTimes(0);
+      expect(setShouldShowTenantPopup).toBeCalledWith(false);
+      done();
+    });
+  });
+
+  it('Should not show tenant selection popup when multitenancy is disabled dynamically', (done) => {
+    const multiTenancyDisabledConfig = {
+      multitenancy: {
+        enable_aggregation_view: true,
```

Comment:
nit: remove this line

Reply:
I think you meant in account-nav-button.test.tsx? But I got your point. Removed it since we are no longer caring about that variable. I also changed the name of the test. This is needed in here to determine what we set as the value on the higher level.